apps/builder/app/builder/features/style-panel/style-source/style-source-input.tsx

```diff
@@ -221,10 +235,20 @@ const markAddedValues = <Item extends IntermediateItem>(
   return items.map((item) => ({ ...item, isAdded: valueIds.has(item.id) }));
 };
 
+const userActionStates = [
+  ":hover",
+  ":active",
+  ":focus",
+  ":focus-visible",
+  ":focus-within",
+];
```

Comment:
list needs discussion

Reply:
Took the list here
https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes#user_action_pseudo-classes

Reply:
Discussed this list with taylor, I think we are fine, the initial idea was to stick with wf naming and initial list, but they have weird names, e.g. they call :active "pressed" or they call :focus-visible "focus keyboard" its clear that they were trying to be better understood by non-developers, but this type of renaming is harmful long-term, we will have an explanation under the  dropdown for each


tldr your list is fine